dependencies.cfg

```diff
@@ -3,4 +3,4 @@
 #   name: dependency name
 #   separator: colon + optional white space
 #   version: git branch or tag
-integrations.library: v0.5.1
+integrations.library: v0.5.3
```

Comment:
Your integrations.library PR has been merged  and I've create a new release: `v0.6.0`
Please update the dependency, this will fix the build errors.